### ISSUE TITLE
Bulletin Board All Items Shown

### ISSUE THREAD

**Issue by Digit** [pioneerspacesim/pioneer#5760]
### Observed behaviour
Bulletin Board is populated with many dozens of missions etc.

### Expected behaviour
Typically only maybe a couple scan missions if even, maybe a dozen or two missions/items max total.

### Steps to reproduce
* Yet to determine.  
** Half notion may relate to save reloading, but that's a tenuously formed notion, may be a red-herring
*** (Marginally less tenuous a notion, when encountering this before, _[(while saves were not retaining shop state from a prior bug(... I thought I knew which to link, but failed to find ... maybe mandela effect, heh.)]_, an exit out and load back in and this could be reverted to a normal bulletin board listing... ~ I think.  not 100% sure about that.  Vague recollection suggests.   But now, I caught a save with that overpopulated bulletinboard, that reloads with the bulletin board still overloaded.)

__My pioneer version (and OS):__
* Pioneer version: Master branch, 48efc4667, 
* OS: Linux 

__My output.txt (required) and game save (optional, but recommended)__
* output: http://ks392457.kimsufi.com/stuff/pioneer/output.txt
* Save with example overpopulated bulletin board at Poseidon station : http://ks392457.kimsufi.com/stuff/pioneer/M20240130%20b%20mola%20hd4%20100

__Screenshots:__
http://ks392457.kimsufi.com/stuff/pioneer/scrots/
I made these with a scrot ; sleep 1 ; scrot ; sleep 1 type of command chain while I scrolled through.  These screenshots give some idea what it's like, though by far not all that's there.  For example, look how small the scrollbar handle is.
![1](http://ks392457.kimsufi.com/stuff/pioneer/scrots/2024-02-11-094944_1920x1080_scrot.png)
![2](http://ks392457.kimsufi.com/stuff/pioneer/scrots/2024-02-11-094948_1920x1080_scrot.png)
![3](http://ks392457.kimsufi.com/stuff/pioneer/scrots/2024-02-11-094952_1920x1080_scrot.png)
![4](http://ks392457.kimsufi.com/stuff/pioneer/scrots/2024-02-11-094956_1920x1080_scrot.png)
![5](http://ks392457.kimsufi.com/stuff/pioneer/scrots/2024-02-11-095000_1920x1080_scrot.png)
![6](http://ks392457.kimsufi.com/stuff/pioneer/scrots/2024-02-11-095003_1920x1080_scrot.png)
![7](http://ks392457.kimsufi.com/stuff/pioneer/scrots/2024-02-11-095007_1920x1080_scrot.png)
![8](http://ks392457.kimsufi.com/stuff/pioneer/scrots/2024-02-11-095010_1920x1080_scrot.png)
![9](http://ks392457.kimsufi.com/stuff/pioneer/scrots/2024-02-11-095014_1920x1080_scrot.png)
![10](http://ks392457.kimsufi.com/stuff/pioneer/scrots/2024-02-11-095019_1920x1080_scrot.png)
![11](http://ks392457.kimsufi.com/stuff/pioneer/scrots/2024-02-11-095023_1920x1080_scrot.png)

__A feature, not a bug.__
When playing, and encountering this on rare occasions, it can be interpreted with a narrative of some event taking place nearby that caused the surge in activity.  Then there's no shattering of suspension of disbelief.   ... Though, it is a stretch when that there are that many an increase over the norm.

**Comment by impaktor** [comment]
Yeah, this is definitely a bug, not a feature. Doesn't look like they're duplicates, so for some reason very many adverts are generated sometimes.

**Comment by rochr4** [comment]
I confirm, orbited nearby planet for days and this awaited me in a Station nearby.

[https://i.imgur.com/8g4SYvX.gif](url)

**Comment by Digit** [comment]
> I confirm, orbited nearby planet for days and this awaited me in a Station nearby.
> 
> [https://i.imgur.com/8g4SYvX.gif](url)

Thanks.  

As amended to original post:

Okay, so understanding the bulletin board starts with few, then accumulates more over time, it stretches the suspension of disbelief in a different way... which would seem less jarring by **keeping the total number of bulletin board items in a range similar to as originally appear when first arriving**, rather than continually accumulating.  

... Does that count as a feature request?  :)

**Comment by Bodasey** [comment]
to mention / to compare #4273 is still kept open (Sorry, I can not run pioneer and test)

**Comment by zonkmachine** [comment]
Number of ads in the bulletin board after some 9 weeks of just sitting still from start at Barnard's star.
------------------------------------------------------
Assassination                 10
Cargo run                       21
Scout                           119 !!!
Combat                            6
Deliver Package             24
Search and rescue           3
Second hand                    2
Taxi                                 34

Maybe, just maybe, the Scout module could chill it a bit. Some balancing needed.

**Comment by zonkmachine** [comment]
I had a go at a quick fix for the Scout mission as it seem a bit more spammy than the rest of the modules. https://github.com/pioneerspacesim/pioneer/pull/6095

**Comment by zonkmachine** [comment]
https://github.com/pioneerspacesim/pioneer/pull/5312 Saw some changes in sorting of the ads. This means that similarly looking ads now end up next to each other. The ads have a limited number of titles available and when you add a lot of missions/ads from the same module, these will form groups of similarly looking titles and when the description lines also look the same, then this is will be repetitious as described in this issue.

Some ways to avoid this is to:
* add more titles/descriptions (n.b. not starting on the same letter)
* Add variation within the titles/descriptions with variables
* Create fewer ads/missions
